src/frontend/query_commands.ml

```diff
@@ -359,6 +359,43 @@ let dispatch pipeline (type a) : a Query_protocol.t -> a =
     in
     List.map ~f:Mbrowse.node_loc path
 
+  | Locate_type pos ->
+    let typer = Mpipeline.typer_result pipeline in
+    let structures = Mbrowse.of_typedtree (Mtyper.get_typedtree typer) in
+    let pos = Mpipeline.get_lexing_pos pipeline pos in
+    let path = Mbrowse.enclosing pos [structures] in
+    let path =
+      Std.List.filter_map path ~f:(fun (env, node) ->
+        Locate.log ~title:"debug" "inspecting node: %s" (Browse_raw.string_of_node node);
```

Comment:
Nitpick: I'd change `"debug"` for `"query_commands Locate_type"` or something of the sort.